src/transformers/modeling_utils.py

```diff
@@ -1164,17 +1167,22 @@ def add(self, hyp, sum_logprobs):
             else:
                 self.worst_score = min(score, self.worst_score)
 
-    def is_done(self, best_sum_logprobs):
+    def is_done(self, best_sum_logprobs, cur_len=None):
```

Comment:
this is a fix: current hypothesis should not be evaluated as if it were of length, `max_len`

Reply:
@thomwolf correct me if I'm wrong :)